dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/net/TracingRequestInterceptor.kt

```diff
@@ -71,9 +72,13 @@ internal class TracingRequestInterceptor(acceptedHosts: List<String> = emptyList
         val span = startedSpans.remove(identifyRequest(request))
         if (span != null) {
             span.setTag(Tags.HTTP_STATUS.key, statusCode)
-            if (statusCode >= 400) {
+            if (statusCode in 400..499) {
                 (span as? MutableSpan)?.isError = true
             }
+            if (statusCode == 404) {
+                // limit spam on 404 resources
```

Comment:
What do you mean by limiting it ?

Reply:
Well if the resource returns 404 it means it's not a valid URL. This can lead to seeing a lot of errors with different urls, when in fact it's all a 404. 
Copied this from `dd-trace-java` actually, it's a rule they use